Source/WebCore/dom/DeviceOrientationAndMotionAccessController.cpp

```diff
@@ -82,7 +82,7 @@ void DeviceOrientationAndMotionAccessController::shouldAllowAccess(const Documen
         if (permissionState != DeviceOrientationOrMotionPermissionState::Granted)
             return;
 
-        for (AbstractFrame* frame = m_topDocument.frame(); frame && frame->window(); frame = frame->tree().traverseNext()) {
+        for (Frame* frame = m_topDocument.frame(); frame && frame->window(); frame = frame->tree().traverseNext()) {
```

Comment:
Can this use `auto*`?

Reply:
This one can't.  Document::frame returns the class that used to be called Frame and is now LocalFrame, but in this loop we need it to be the class that used to be called AbstractFrame and is now Frame.